python-client/tests/fixtures/always_wrong.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import pytest
+from giskard.ml_worker.core.giskard_dataset import GiskardDataset
+from giskard.ml_worker.core.model import GiskardModel, logger
+import numpy as np
+from sklearn.dummy import DummyClassifier
+from tests import path
+
+input_types = {
+    "input": "numeric",
+    "sex": "category"
+}
+
+
+@pytest.fixture()
+def always_wrong_data() -> GiskardDataset:
+    logger.info("Reading always_wrong.csv")
+    df = pd.read_csv(path("test_data/always_wrong.csv"), keep_default_na=False, na_values=["_GSK_NA_"])
+    return GiskardDataset(
+        df=df,
+        column_types=df.dtypes.apply(lambda x: x.name).to_dict(),
+        target="output",
+        feature_types=input_types
+    )
+
+
+@pytest.fixture()
+def always_wrong_model() -> GiskardModel:
```

Comment:
I'd put a dummy model fixture to german_credit_scoring.py since "always wrong" is a classification concept, in regression there's no binary distinction between "wrong" and "right".
Also, since we're in the context of german_credit_scoring dataset, instead of `always_wrong` I'd call it `always_default`.
Then in your test you could pass a reference slice as:
```
reference_slice = data.slice(lambda df: df[(df.sex == "female") & (df.default == "Not default")])
```

in this case you'll know that the true value of every reference slice row is "Not default" but your dummy model always returns "Default" so the metric will be 0